app/controllers/admin/users_controller.rb

```diff
@@ -46,12 +46,11 @@ def show
   end
 
   def delete_all_posts
```

Comment:
I wonder if we should also rename (or better deprecate) this methods since it's not deleting all posts anymore?